examples/README.md

```diff
@@ -6,9 +6,9 @@ This directory contains example apps for each framework we support.
 
 1. Create or Update an example at [`examples/{next,vue,angular}/...`](examples)
 
-   For your `aws-exports.js`, you can [reference an existing or create a new environment](environments).
+   For your `aws-exports.js`, you can [reference an existing or create a new environment](../environments/README.md).
 
-1. Run your example at monorep root: `yarn {next,vue,angular}-example dev`
+1. Run your example at monorep root: `yarn {react,vue,angular}-example dev`
```

Comment:
Actually, we don't want to change this because the react example uses next, so it is `yarn next-example dev`
```suggestion
1. Run your example at monorep root: `yarn {next,vue,angular}-example dev`
```

Reply:
so...should we write both `react` and `next` in the README since it's easier for users to pick up which ever makes more sense to them?

As a new contributor, I felt `next` was confusing to me since we don't have a `nextjs` switcher on top of the docs.

And I found `react` works, but was not added to the README, which can be confusing as well

Reply:
idk if we need to say `next` at all, since it's more of internal tooling. `next` is to used to setup `react` example, but the focus is still `react`.

Reply:
100% agree with @wlee221 that it makes more sense. I didn't realize we already had this command.